wepay/waltz
# ducktape test - client disconnected from server

WaltzClient establishes a network connection with Waltz Server to send/receive transactions. This test simulates network disconnection between client and server.

How to test it:
add print statements connection_interruption.py to see when command to drop/allow traffic through a server port is fired. With this knowledge check the metrics output of server and storage nodes to see that transactions aren't going through. 
Further testing can me done with tcpdump port 6060/7070 or by checking iptables
